amundsen_application/static/webpack.common.ts

```diff
@@ -70,8 +76,11 @@ const config: webpack.Configuration = {
       ],
     },
     plugins: [
+      new CleanWebpackPlugin(),
```

Comment:
My understanding is that `CleanWebpackPlugin` removes things in `dist/`, but I think some part of our build script in the private repo leverages the `dist/` folder. I'm not recalling enough context at the moment to know if this is an issue, but could you take a look at the `partnerized-static-build.sh` to see what I'm referencing. However if all of the commands and and flags still work in that script with these changes then I assume we're in the clear.

Reply:
Yes, this plugins cleans up the /dist folder before creating a new version. 

Hmmm... I see in the script that most of the times we won't copy over the /dist folder, and we will build it again. It is working fine on dev for now, I have tried different runs of the script as well.

I will keep an eye on it just in case, and I will test it on staging before merging.

Thanks for the heads up!